main.py

```diff
@@ -16,13 +16,11 @@
 import tensorflow as tf
 from loguru import logger
 
-import constants
-import contributivity
-import multi_partner_learning
-import scenario
-import utils
+from subtest import constants
```

Comment:
what is `subtest` ?

Reply:
it is the name of the package (temporary). At the end, the library would be installed via pip

 pip install subtest